lib/common.sh

```diff
@@ -946,11 +946,15 @@ common::set_cloud_binaries () {
 
   if [[ -x "$GSUTIL" && -x "$GCLOUD" ]]; then
     logecho -r $OK
-    return 0
   else
     logecho -r $FAILED
     return 1
   fi
+
+  # 'gcloud docker' access is now set in .docker/config.json
+  logrun $GCLOUD --quiet auth configure-docker || return 1
```

Comment:
will the rest of this change work without this line? this still seems to be problematic for CI.
cc @dekkagaijin who I think was discussing this in #bazel just last week

Reply:
There's a(nother) bug with `gcloud auth configure-docker`, turns out that `docker version` returns helpful info and a non-zero exit code when it can't talk to `dockerd`... who knew.